components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/ActionExecutorServiceImpl.java

```diff
@@ -127,6 +152,17 @@ public ActionExecutionStatus execute(ActionType actionType, String[] actionIdLis
             return execute(action, eventContext);
         } catch (ActionExecutionRuntimeException e) {
             // todo: add to diagnostics
+            if (LoggerUtils.isDiagnosticLogsEnabled()) {
+                DiagnosticLog.DiagnosticLogBuilder diagLogBuilder = new DiagnosticLog.DiagnosticLogBuilder(
+                        ActionExecutionConstants.LogConstants.ACTION_EXECUTION,
+                        ActionExecutionConstants.LogConstants.ActionIDs.EXECUTE_ACTION);
+                diagLogBuilder
+                        .resultMessage("Skip executing actions for " + actionType + " type.")
```

Comment:
Since this method runs a specific action let's mention which action is skipped. We can use the name and the id of the action

Reply:
Updated in